src/otx/algorithms/visual_prompting/adapters/openvino/model_wrappers/openvino_adapters.py

```diff
@@ -0,0 +1,164 @@
+"""Openvino Adapter Wrappers of OTX Visual Prompting.
+
+There is a bug on fit_to_window resize module in model API.
+VisualPromptingOpenvinoAdapter is temporarily implemented to use updated `fit_to_window` resize function.
+When model API version in otx is upgraded, it can be removed.
+
+Issue: https://github.com/openvinotoolkit/model_api/issues/99
+Updated PR: https://github.com/openvinotoolkit/model_api/pull/100
+"""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+
+from functools import partial
+from typing import Tuple
+
+import numpy as np
+import openvino.runtime as ov
+from openvino.model_api.adapters import OpenvinoAdapter
+from openvino.preprocess import ColorFormat, PrePostProcessor
+from openvino.runtime import Output, Type
+from openvino.runtime import opset10 as opset
+from openvino.runtime.utils.decorators import custom_preprocess_function
+
+
+def resize_image_with_aspect_pad(input: Output, size, keep_aspect_ratio, interpolation, pad_value):
```

Comment:
@jaegukhyun Do you agree with this comment?
If so, I'll move it and create temporary adapter, then any task that needs this custom patch can use it.
@sungmanc Could you recommend the location?

Reply:
I agree on this function needs to be moved to more general section, but it's over the scope of this PR. I think additional PR would be better, because changing ops for overall task should be more validated.